### ISSUE TITLE
Should this line trigger some sort of linting error?

### ISSUE THREAD

**Issue by asottile** [PyCQA/pycodestyle#631]
I had a hard time finding something in pep8 that would forbid this (and pycodestyle itself doesn't trigger a lint) but deep down inside I feel it surely should :)

```python
[(a,)for a in y]
```

Maybe something like "expected spaces around keyword `for`"?

**Comment by sigmavirus24** [comment]
> but deep down inside I feel it surely should

I feel the same.

This should probably be a new check altogether though.

**Comment by jayvdb** [comment]
This is a grey area as I noted in a similar case https://github.com/PyCQA/pycodestyle/issues/570

> But [pep 8] doesnt address when unassigned data-structures are used immediately near a keyword like `if` or `for`.

I suspect this should be a non-controversial update to PEP 8.

**Comment by asottile** [comment]
Another (related) strange one:

```python
from .import import bar
```
(works the same as `from . import bar`)

**Comment by FichteFoll** [comment]
Is that supposed to be `from .import bar`?

**Comment by asottile** [comment]
yep, woops, edited

**Comment by hoylemd** [comment]
Would the rule here be 'all keywords must be preceded and followed by one space character'?

**Comment by asottile** [comment]
Another related case:

```python
try:
    ...
except(IOError,):
    ...
```

**Comment by asottile** [comment]
Duplicate of #371
